src/pages/Offer/sections/InsuredAmount.tsx

```diff
@@ -52,37 +53,45 @@ const InfoText = styled('div')({
   display: 'inline',
 })
 
-const rows: ReadonlyArray<{
+const rows = (
+  isStudent: boolean,
+): ReadonlyArray<{
   titleKey: string
   amountKey: string
-}> = [
+}> => [
   {
     titleKey: 'OFFER_INSURED_AMOUNT_COL_ONE_TITLE',
     amountKey: 'OFFER_INSURED_AMOUNT_COL_ONE_AMOUNT',
   },
   {
     titleKey: 'OFFER_INSURED_AMOUNT_COL_TWO_TITLE',
-    amountKey: 'OFFER_INSURED_AMOUNT_COL_TWO_AMOUNT',
+    amountKey: isStudent
+      ? 'OFFER_INSURED_AMOUNT_COL_TWO_AMOUNT_STUDENT'
+      : 'OFFER_INSURED_AMOUNT_COL_TWO_AMOUNT',
   },
   {
     titleKey: 'OFFER_INSURED_AMOUNT_COL_THREE_TITLE',
-    amountKey: 'OFFER_INSURED_AMOUNT_COL_THREE_AMOUNT',
+    amountKey: isStudent
+      ? 'OFFER_INSURED_AMOUNT_COL_THREE_AMOUNT_STUDENT'
+      : 'OFFER_INSURED_AMOUNT_COL_THREE_AMOUNT',
   },
   {
     titleKey: 'OFFER_INSURED_AMOUNT_COL_FOUR_TITLE',
     amountKey: 'OFFER_INSURED_AMOUNT_COL_FOUR_AMOUNT',
   },
 ]
 
-export const InsuredAmount: React.SFC = () => (
+export const InsuredAmount: React.SFC<{ insuranceType: InsuranceType }> = ({
+  insuranceType,
+}) => (
   <Card>
     <HeaderWrapper>
       <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_TITLE">
         {(title) => <Header>{title}</Header>}
       </TranslationsConsumer>
     </HeaderWrapper>
     <Table>
-      {rows.map((row, index) => (
+      {rows(isStudentInsurance(insuranceType)).map((row, index) => (
```

Comment:
📸

Reply:
?

Reply:
I knew this was a little too unclear. I wanted to convey that it was clean enough to be picture-worthy 🤷‍♂️

Reply:
haha right 😄